### DISCUSSION TITLE
py.typed missing when installing poetry dependencies from Git

### DISCUSSION THREAD

**Discussion by krishan-patel001** [python-poetry/poetry#8598] [Q&A]
Hi,
I have a project lets call it `projectA` and I want to import this project and use it in `projectB` however when I perform `poetry add git+https://github.com/org/projectA` in `projectB` the `py.typed` file is not in the site-packages of `projectB`

This is the tree structure of `projectA`
```
├── Makefile
├── README.md
├── poetry.lock
├── pyproject.toml
├── src
│   └── projectA
│       ├── __init__.py
│       └── py.typed
└── tests
```
And this is what the .venv folder looks like in `projectB` once I run the poetry add command described above. 
```
.venv
├── bin
├── lib
│   └── python3.9
│       └── site-packages
│           ├── projectA
│           │   └── __init__.py
```

So I am wondering where the `py.typed` file has gone and why it is not in the site-packages. It is worth noting that I previously used setup tools and did ran pip install with the same git link and it worked fine and the `py.typed `was in the site-packages folder.

Any help would be appreciated.

**Comment by dimbleby** [comment]
does not reproduce:
```console
(foo) $ poetry init -n
(foo) $ poetry add git+https://github.com/python-poetry/poetry-core.git

Updating dependencies
Resolving dependencies... (0.1s)

Package operations: 1 install, 0 updates, 0 removals

  • Installing poetry-core (1.8.1 638fc58)

Writing lock file
(foo) $ ls /home/dch/.virtualenvs/foo/lib/python3.10/site-packages/poetry/core/py.typed
/home/dch/.virtualenvs/foo/lib/python3.10/site-packages/poetry/core/py.typed
```
hard to guess what you've misunderstood

**Reply by krishan-patel001** [reply]
Hi, sorry it turns out I had a syntax error in my pyproject.toml which was causing it to build incorrectly. all sorted now.
